services/src/main/java/org/exoplatform/news/rest/NewsTargetingEntity.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
this license header to be used for exoplatform https://github.com/exoplatform/news/pull/344/files#diff-86d9ffd37e7cbf92265fc84802a41571c0af4c8ce123face695b139272f4bfd2R1-R16